.github/workflows/ci.yaml

```diff
@@ -0,0 +1,113 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - master
```

Comment:
There is no `master` branch in this repository.

Reply:
Fixed in 814080e6.